homework3/task04/tests/test_is_armstrong.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from project_is_armstrong.task4_armstrong import is_armstrong
+
+
+@pytest.mark.parametrize("number", [153])
+def test_is_armstrong_true(number: int):
+    assert is_armstrong(number) is True, "Is Armstrong number"
```

Comment:
Can you check the message with `, "Is Armstrong number"` and without?

Please put both variants to the comment.

Reply:
Yes, you are right. If test does not pass, this text of error will occur. But we in advance know that test will pass here , thus we dont need this text of error.